easybuild/tools/options.py

```diff
@@ -375,6 +375,7 @@ def override_options(self):
                           None, 'store', None, 'e', {'metavar': 'CLASS'}),
             'enforce-checksums': ("Enforce availability of checksums for all sources/patches, so they can be verified",
                                   None, 'store_true', False),
+            'env-for-shebang': ("Define the env command to use when fixing shebangs", None, 'store', '/usr/bin/env'),
```

Comment:
@mboisson You should use the `DEFAULT_ENV_FOR_SHEBANG` constant here to specify the default value, not hardcode `/usr/bin/env`